src/core/probe/kernel/mod.rs

```diff
@@ -344,6 +349,14 @@ impl Kernel {
 
         Ok(desc)
     }
+
+    /// Enable libbpf-rs debugging.
+    pub(crate) fn debug(&mut self, debug: bool) -> Result<()> {
```

Comment:
The new way of initializing modules _before_ we've parsed the arguments make this difficult. So, either we punt a whole in the API at `Kernel::attach()` or we add a new method.
I've chosen to do neither of those because they seem to me as dirtying a nice and clean interface. So I've chosen to use OnceCell. Comments?